recipes/lodei/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: lodei
+  version: 1.0.0
+
+source:
+  url: https://github.com/rna-editing1/lodei/archive/refs/tags/v1.0.0.tar.gz
+  md5: aab7ea1fa9c1e7b250b8b2d02065dd5d
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+  run_exports:
+    - {{ pin_subpackage('lodei', max_pin="x") }}
```

Comment:
Do you assume that every minor version 1.1 or 1.1.1 will break the CLI?

Reply:
I don't assume that minor version will break the CLI.